templates/phoenix_elixir/files/README.md

```diff
@@ -0,0 +1,29 @@
+# Phoenix for Platform.sh
+
+<p align="center">
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/phoenix_elixir/.platform.template.yaml&utm_content=phoenix_elixir&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+This template builds a simple application using Phoenix.
+
+Phoenix is a web framework written in Elixir that implements the server-side MVC pattern, prioritizing speed and concurrency.
```

Comment:
That may be how they describe themselves, but there is no such thing as server-side MVC.  It doesn't exist. Don't perpetuate that lie...